pkg/apis/hco/v1beta1/hyperconverged_types.go

```diff
@@ -234,9 +234,13 @@ type OperandResourceRequirements struct {
 // +k8s:openapi-gen=true
 type HyperConvergedObsoleteCPUs struct {
 	// MinCPUModel is the Minimum CPU model that is used for basic CPU features; e.g. Penryn or Haswell
```

Comment:
Penryn or Haswell.
Please add a dot here to make the CRD nicer.

Reply:
Done